Patterns/Japanese/Japanese-Numbers.yaml

```diff
@@ -1,11 +1,6 @@
 ﻿---
 #JapaneseNumberParserConfiguration
 LangMarker: 'Jpn'
-DecimalSeparatorChar: !char '.'
```

Comment:
It's best not to delete items from the YAML file, unless really necessary.
The code later may try to use them and break. Also, in the ideal case, the YAML files between languages should be as similar as possible.